app/controllers/projects/samples/metadata_controller.rb

```diff
@@ -22,6 +22,30 @@ def edit
                                                    value: params[:value]
                                                  }), status: :ok
       end
+
+      def destroy
+        authorize! @project, to: :update_sample?
```

Comment:
You can remove this authorization here since the `Samples::Metadata::UpdateService` handles that in the `execute`

Reply:
Removed in 969c18a281013c8d4311e2b0ddb355e3066185f6